kernelci/lava.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2019 Linaro Limited
+# Author: Dan Rue <dan.rue@linaro.org>
+#
+# This module is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+
+def get_device_type_by_name(name, device_types, aliases=[]):
```

Comment:
I wasn't sure where to stick this function. I had to put it in lib/ or kernelci/ to be able to write tests against it.

Reply:
That's fine, the scripts to generate and submit jobs need to be consolidated in the same way as `kci_build`.  So yes `kernelci.lava` sounds like a logic place to store all things LAVA-related.